embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/logging/ReportingLoggerAction.kt

```diff
@@ -26,8 +23,9 @@ internal class InternalErrorLogger(
         if (finalThrowable != null) {
             try {
                 internalErrorService.handleInternalError(finalThrowable)
-            } catch (exc: Exception) {
-                logger.log(exc.localizedMessage ?: "", InternalEmbraceLogger.Severity.ERROR, null, false)
+            } catch (e: Throwable) {
+                // Yo dawg, I heard you like to handle internal errors...
+                Log.w(EMBRACE_TAG, msg, e)
```

Comment:
The pipe to the sdk is broken so lets just log an error to logcat and move one. We should do better when we redo the internal error service but now is not that time.